src/main/perl/MD.pm

```diff
@@ -55,10 +56,11 @@ sub _initialize
 
     my $st = $config->getElement($path)->getTree;
     $path=~m!/([^/]+)$!;
-    $self->{devname} = $1;
+    $self->{devname} = unescape($1);
```

Comment:
Only when you specify your device as md/NAME instead of mdXX, it needs to be escaped in the templates/profiles. In my test it also works with an old entry, i.e. md0, which is not escaped.